src/pages/find-offers-page/FIndOffers.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react'
+import { styles } from '~/containers/guest-home-page/how-it-works/HowItWorks.styles'
+import { useTranslation } from 'react-i18next'
+import AppSwitcher from '~/components/app-content-switcher/AppContentSwitcher'
+
+const FIndOffers = () => {
```

Comment:
Typo FIndOffers, second letter
Name file the same

Reply:
fixed